_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/consume-topic-key-value.adoc

```diff
@@ -4,13 +4,13 @@ View the messages being produced to the Kafka topic in Confluent Cloud.
 There are many ways to do this, including the Confluent Cloud UI, but for this tutorial we will show you how to it with the Confluent Cloud CLI.
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/harness-console-consumer-keys.sh %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/harness-console-consumer-keys.sh %}</code></pre>
```

Comment:
Follwing these instructions gave me this error:

Error: no API key selected for resource "lkc-yjrwp"

Suggestions:
    Select an API key for resource "lkc-yjrwp" with `ccloud api-key use <API_KEY> --resource lkc-yjrwp`.
    To do so, you must have either already created or stored an API key for the resource.
    To create an API key, use `ccloud api-key create --resource lkc-yjrwp`.
    To store an existing API key, use `ccloud api-key store --resource lkc-yjrwp`.

Should we instruct the user on the `ccloud api-key store` and `ccloud api-key use` steps to make the `ccloud kafka topic consume` work?

Reply:
This should be covered by http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/merge_ccloud_datagen/kafka-connect-datagen/confluent.html#download-and-setup-the-confluent-cloud-cli